snippets/multisign.py

```diff
@@ -57,11 +58,9 @@
 if multisigned_tx_response.result["validated"]:
     print("The multisigned transaction was accepted by the ledger:")
     print(multisigned_tx_response)
-    if multisigned_tx_response.result["Signers"]:
-        print(
-            "The transaction had "
-            f"{len(multisigned_tx_response.result['Signers'])} signatures"
-        )
+    signers_in_response = multisigned_tx_response.result["tx_json"]["Signers"]
+
+    print("The transaction had " f"{len(signers_in_response)} signatures")
```

Comment:
You still need the `if signers_in_response` that existed previously in the `if`

Reply:
thanks, fixed in [376d823](https://github.com/XRPLF/xrpl-py/pull/720/commits/376d823fc7e88f14b2d8a7f398a71fbaa6bfc8c1)